06-11-2018/Italent2BoardCreation/src/main/resources/Application.properties

```diff
@@ -0,0 +1,4 @@
+posturl=https://italent2.demo.lithium.com/restapi/vc/authentication/sessions/login?restapi.response_format=json&user.login=navyakeerthi&user.password=Q!w2e3r4
```

Comment:
all psoturl, boardurl, messageurl,replyurl url path is same .so create one property "https://italent2.demo.lithium.com/restapi/vc/" remaining every thing modify in code